Source/WebCore/rendering/RenderCombineText.h

```diff
@@ -53,6 +54,7 @@ class RenderCombineText final : public RenderText {
     float m_combinedTextWidth { 0 };
     float m_combinedTextAscent { 0 };
     float m_combinedTextDescent { 0 };
+    String m_renderingText;
```

Comment:
```suggestion
    String m_renderedText;
```

I think this name would match better the webkit style guide.

Reply:
Will update. After EWS run.  Thanks!

Reply:
Fixed.